ostree-ng.sh

```diff
@@ -481,16 +494,133 @@ greenprint "🧹 Clean up compose"
 sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete installer > /dev/null
 
+# Ensure SELinux is happy with our new images.
+greenprint "👿 Running restorecon on image directory"
+sudo restorecon -Rv /var/lib/libvirt/images/
+
+##################################################################
+##
+## Install edge vm with edge-installer (http boot)
+##
+##################################################################
+
+greenprint "📋 Mount installer iso and copy content to webserver/httpboot"
+sudo mkdir -p ${HTTPD_PATH}/httpboot
+sudo mkdir -p /mnt/installer
+sudo mount -o loop "/var/lib/libvirt/images/${ISO_FILENAME}" /mnt/installer
+sudo cp -R /mnt/installer/* ${HTTPD_PATH}/httpboot/
+sudo chmod -R +r ${HTTPD_PATH}/httpboot/*
+sudo umount --detach-loop --lazy /mnt/installer
+# Remove mount dir
+sudo rm -rf /mnt/installer
+
+# Create new kickstart file for httpboot
+# sudo tee ${HTTPD_PATH}/ks.cfg > /dev/null << EOF
+sudo tee "$KS_FILE" > /dev/null << EOF
+text
+network --bootproto=dhcp --device=link --activate --onboot=on
+zerombr
+clearpart --all --initlabel --disklabel=msdos
+autopart --nohome --noswap --type=plain
+ostreesetup --osname=rhel --url=http://192.168.100.1/httpboot/ostree/repo --ref=$OSTREE_REF --nogpg
+user --name=mcattamo --groups=wheel --iscrypted --password="\$6\$1LgwKw9aOoAi/Zy9\$Pn3ErY1E8/yEanJ98evqKEW.DZp24HTuqXPJl6GYCm8uuobAmwxLv7rGCvTRZhxtcYdmC0.XnYRSR9Sh6de3p0"
```

Comment:
Please do not use your name in test case. Thanks.

Reply:
Ok I will replace it with httpuser.